### DISCUSSION TITLE
Tab rows not scrollable

### DISCUSSION THREAD

**Discussion by leochan369** [onemen/TabMixPlus#332] [General]
I successfully installed TabMixPlus today with the latest version.  I have been waiting for this to come for a long time!

However, I found that I am not able to scroll through the tab rows with the mouse wheel.  

After playing around with the settings, I found that the only way to make it work is to click on the very thin scroll bar and drag it up and down.  That space is about 2 pixels wide...

Is this something that will be fixed in the near future?

**Comment by onemen** [comment]
What is your Firefox version?
Try 
https://github.com/onemen/TabMixPlus/releases/tag/dev-build

post a screenshot
attach a file with your Tab mix preferences

**Reply by leochan369** [reply]
I am using 131.0, it just automatically updated yesterday, so I suppose that's the latest version.  Settings attached.

I will try the dev build later on... when work is less busy.

Thank you very much
![Tab Mix Plus settings](https://github.com/user-attachments/assets/4f91b0ab-53a8-4fe7-adba-caa58f303c8d)

**Comment by onemen** [comment]
Attach a file with your Tab mix plus preferences:
Open Tab Mix Plus Options and click the 'Preferences' button at the bottom left corner.
Choose 'Export Preferences' and save the file to a location on your hard drive. Drag the file into new reply to attach it.


>After playing around with the settings, I found that the only way to make it work is to click on the very thin scroll bar and drag it up and down. That space is about 2 pixels wide
>

Post a scrrenshoot of your browser, that show what exectly you click

**Reply by leochan369** [reply]
Both files are now attached.
[TMPpref.txt](https://github.com/user-attachments/files/17247252/TMPpref.txt)
![Tab Mix Plus Scrollbar](https://github.com/user-attachments/assets/23be5196-93e5-4ff7-926f-7b50dfa86422)

**Reply by leochan369** [reply]
Also same problem persists with Dev build.

Thank you very much for your help!

**Comment by onemen** [comment]
Do you see any messages in the brwoser console?
Try to create new profile. See our docs, how to install tabmix an firefox scripts

**Reply by leochan369** [reply]
I actually installed this version following your documentation, installing the latest firefox scripts and the latest version of Tab Mix Plus (and the Dev build too)

Sorry - how do I call out the browser console please?

**Reply by onemen** [reply]
Look in tools menu for the browser console.

![image](https://github.com/user-attachments/assets/4a6a4e6f-6c25-4c8a-bce0-ed1fd93108bd)
I have no idia where this button come from
Are you using other extensions

**Comment by leochan369** [comment]
This is a screenshot of the browser console.  Is there any lines I should copy to here?
![image](https://github.com/user-attachments/assets/81e2c17d-fb18-4eec-9560-cdff65547985)

That's not a button, it is a scroll bar.  By clicking this and dragging it up and down, I am able to see the rows of tabs - I have about 10 rows of tabs in Firefox currently.

**Reply by onemen** [reply]
click on this.window.gBrowserInit and post a screenshot with its content.

Are you using any other extensions?

your Tab mix preference is not set to use multy-row, how do you have 10 rows?

**Comment by leochan369** [comment]
Yes that's it!  I didn't set it as multirow.  I think the multirow occurs because there was another plugin that I used before.

Thank you so much for your help!  Problem solved.

**Comment by mscir** [comment]
Thanks very much, I had the same problem, 4 rows of tabs but I had not set TMP to multi-row. Cheers.

**Comment by mscir** [comment]
Thanks very much, I had the same problem, 4 rows of tabs but I had not set TMP to multi-row. Cheers.
